From 6a9091a27db09f018ff96fa4c4d2913bb65b827a Mon Sep 17 00:00:00 2001
From: WingMan Kwok <w-kwok2@ti.com>
Date: Wed, 18 May 2016 08:31:31 -0400
Subject: [PATCH 23/65] ti_config_fragments/connectivity.cfg: enable keystone
 netcp cpts support

Enable support of cpts for both gbe and 10gbe on keystone
devices.

Signed-off-by: WingMan Kwok <w-kwok2@ti.com>
Signed-off-by: Jacob Stiffler <j-stiffler@ti.com>
---
 ti_config_fragments/connectivity.cfg | 1 +
 1 file changed, 1 insertion(+)

diff --git a/ti_config_fragments/connectivity.cfg b/ti_config_fragments/connectivity.cfg
index 43a9b12..e116281 100644
--- a/ti_config_fragments/connectivity.cfg
+++ b/ti_config_fragments/connectivity.cfg
@@ -161,6 +161,7 @@ CONFIG_NET_SCH_FIFO=y
 CONFIG_IP_SCTP=m
 CONFIG_VLAN_8021Q=m
 CONFIG_IP_MULTICAST=y
+CONFIG_TI_KEYSTONE_NETCP_CPTS=y
 
 #Networking drivers
 CONFIG_NET_VENDOR_TI=y
-- 
1.9.1

